### PR TITLE
Describe that fmi2Reset can be called in case of fmi2Error

### LINKED ISSUE #1946: Backport #1940 to FMI 2.0

### PR DESCRIPTION
**Pull request by chrbertsch**
resolves #1946

### CODE REVIEW

File: docs/2_1_common_api.adoc [hunk 1]
```diff
@@ -291,11 +291,12 @@ mode (`loggingOn = fmi2True`).
...
 - `fmi2Error` - the FMU encountered an error.
-The simulation cannot be continued with this FMU instance.
+The simulation cannot be continued.
 If one of the functions returns `fmi2Error`,
 it can be tried to restart the simulation from a formerly stored FMU state by calling `fmi2SetFMUstate`.
 This can be done if the capability flag `canGetAndSetFMUstate` is true and
 `fmi2GetFMUstate` was called before in non-erroneous state.
+It is also possible to reset the FMU by calling `fmi2Reset`.
 footnote:[Typically, `fmi2Error` return is for non-numerical reasons, like "disk full".
 There might be cases where the environment can fix such errors (eventually with the help oft the user),
```
Comment (Maplesoft-fmigroup) [inline docs/2_1_common_api.adoc:301]: oft -> of

File: docs/2_1_common_api.adoc [hunk 2]
```diff
@@ -291,11 +291,12 @@ mode (`loggingOn = fmi2True`).
 Otherwise, `logger` should not show a message.
 
 - `fmi2Error` - the FMU encountered an error.
-The simulation cannot be continued with this FMU instance.
+The simulation cannot be continued.
 If one of the functions returns `fmi2Error`,
 it can be tried to restart the simulation from a formerly stored FMU state by calling `fmi2SetFMUstate`.
 This can be done if the capability flag `canGetAndSetFMUstate` is true and
 `fmi2GetFMUstate` was called before in non-erroneous state.
+It is also possible to reset the FMU by calling `fmi2Reset`.
```
Comment (pmai) [inline docs/2_1_common_api.adoc:299]:
```suggestion
It is also possible to reset the instance of the FMU by calling `fmi2Reset`.
```

**Review by pmai — CHANGES_REQUESTED** [review]
Looks good, just keeping the instance wording seems like a good idea to not suggest that the whole FMU is reset. Should also fix the oft autocorrect typo while we are at it.

**Comment by t-sommer** [comment]
@pmai, can you please re-review?

**Review by pmai — APPROVED** [review]

**Pull request merged**